gateway/proxy/proxy.log

```diff
@@ -0,0 +1,173 @@
+entrypoint.sh
```

Comment:
this file doesn't need to be committed, right?

Reply:
Yes, it was for reference.  I take out.

Reply:
Right.  I put it just for reference.  I take it out.  Thanks!